### PR TITLE
Use specific dir to save temp files during listens dump

### PR DESCRIPTION
**Pull request by paramsingh**
# Summary

* This is a…
    * ( ) Bug fix
    * ( ) Feature addition
    * ( ) Refactoring
    * ( ) Minor / simple change (like a typo)
    * (x) Other

# Problem

We kept running out of space on lemmy while creating full dumps.

# Solution

Change the code to use a configurable directory to save temp files instead of using `/tmp`
by default.

# Action

Merge, merge https://github.com/metabrainz/docker-server-configs/pull/80 and then deploy.

### CODE REVIEW

File: consul_config.py.ctmpl [hunk 1]
```diff
@@ -8,6 +8,8 @@
     {{- end -}}
 {{- end -}}
 
+import os
```
Comment (mayhem) [inline consul_config.py.ctmpl]: Why is this needed? a leftover?

File: listenbrainz/config.py.sample [hunk 2]
```diff
@@ -149,3 +151,5 @@ MAIL_FROM_DOMAIN = 'listenbrainz.org'
 # expiration of 'Remember me' cookie
 SESSION_REMEMBER_ME_DURATION = 365
 
+# listen dumps creation dir
+LISTEN_DUMP_TEMP_DIR_ROOT = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', 'tmp')
```
Comment (mayhem) [inline listenbrainz/config.py.sample]: Ah, I see. Copypasta. :)

**Comment by paramsingh** [comment]
>Will you have a chance to deploy this?

Yep, I'll do it today.
